app/scripts/components/table/list-view.js

```diff
@@ -144,7 +143,7 @@ class List extends React.Component {
       page: this.state.page,
       order: this.state.order,
       sort_by: this.getSortProp(this.state.sortIdx),
-      params: this.state.params,
+      ...this.state.params,
```

Comment:
Are these equivalent? The former operation would place the contents of `this.state.params` on the `params` property, whereas the spread operation will put the contents of `this.state.params` directly on the object.